ArcGISRuntimeSDKQt_CppSamples/Routing/RouteAroundBarriers/RouteAroundBarriers.cpp

```diff
@@ -119,20 +119,18 @@ void RouteAroundBarriers::setMapView(MapQuickView* mapView)
 
 void RouteAroundBarriers::connectRouteSignals()
 {
-  connect(m_routeTask, &RouteTask::createDefaultParametersCompleted, this, [this](const QUuid&, const RouteParameters& defaultParameters)
-  {
-    m_routeParameters = defaultParameters;
-
-    // set flags to return stops and directions
-    m_routeParameters.setReturnStops(true);
-    m_routeParameters.setReturnDirections(true);
-  });
-
   connect(m_routeTask, &RouteTask::doneLoading, this, [this](const Error& loadError)
   {
     if (loadError.isEmpty())
```

Comment:
Same comment as before, and I won't keep repeating it to save time reading through PR comments. Invert these errors checks from "if no error, do this stuff" to "if error, return early". It makes reading the code much easier IMO since the logic doesn't have an additional level of indenting and braces.

Reply:
Modified wherever required.